packages/config-service/src/services/validationService.ts

```diff
@@ -0,0 +1,95 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { GlobalConfig } from './globalConfig';
+
+export class ValidationService {
+  static startUp(envs: NodeJS.Dict<string>): void {
+    // make sure that mandatory fields are passed
+    Object.entries(GlobalConfig.ENTRIES).forEach(([entryName, entryInfo]) => {
+      if (entryInfo.required) {
+        if (!envs[entryName]) {
```

Comment:
Is an existing environment variable with no value going to pass this check?  Do we need to test for something like:
```
value === undefined || value === ''
```

Reply:
Fixed.